recipes/quetz-client/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "quetz-client" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/TheSnakePit/quetz/archive/client_v{{ version }}.tar.gz
+  sha256: 0219fb1f9c2904eb92dcf7ecfbeebefe6f16e1a71f6b486fd67f4ffd0cd68672
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - quetz-client = quetz_client:main
+  script: {{ PYTHON }} -m pip install ./quetz_client --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+
+  run:
+    - python >=3.6
+    - requests >=2,<3
+    - conda-verify >=3,<4
+    - conda-build >=3,<4
+
+test:
+  imports:
+    - quetz_client
+
```

Comment:
you can also add a command-test, e.g. executing

```
quetz-client -h
```


by adding 

```
commands:
  - quetz-client -h
```

Reply:
Adding it, thanks.